### PR TITLE
Add metadata needed for tooling of service reference features

### PR DESCRIPTION
**Pull request by dougbu**
- #10789

### CODE REVIEW

**Comment by dougbu** [comment]
@BillHiebert I made a couple of space changes to the OpenApiItemsSchema.xaml file you provided and added the words "file" or "project" in a couple of spots. Also removed the conditions and the indirections. Is there anything I should test locally before we check this in?

File: src/Mvc/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project>
+  <ItemGroup>
```
Comment (BillHiebert) [inline src/Mvc/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets]: @dougbu  Will this cause the schema file to be imported twice? Are both of these targets files imported?

Comment (dougbu) [reply src/Mvc/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets]: No double import. The `build` targets file is imported only in inner builds i.e. when `TargetFrameworks` isn't set and the `buildMultiTargeting` one is imported only in outer builds. Basically the duplication ensures the items show up no matter what &hellip but only once 😄

**Comment by dougbu** [comment]
Also kicked off a trial official build: https://dnceng.visualstudio.com/internal/_build/results?buildId=221516

**Comment by dougbu** [comment]
@anurse we have a lot of `ServerTests` issues in aspnet/AspNetCore-Internal. But, `Server_MultipleOutstandingSyncRequests_Success` isn't specifically mentioned. Is it likely that Helix failure on Windows 10 AMD64 is just another example of flakiness in the test class?

**Comment by dougbu** [comment]
Sorry to the many people now subscribed to this PR. GitHub added loads of code owners when I changed the target branch ☹️ I've pared things back again.

**Comment by anurse** [comment]
@dougbu Yes, there are a few of these TaskCancelledException failures already tracked in RaaS. I think this is just the same.

**Comment by dougbu** [comment]
Ping reviewers. All checks have passed.

**Review by BillHiebert — APPROVED** [review]

**Pull request merged**